management_service/app/routers/stock.router.ts

```diff
@@ -128,8 +139,88 @@ stockRouter.post("/", async (req: Request, res: Response) => {
       }
     });
     
+    
     console.log("Ordered stock with ID:", stock.stockid);
     res.status(201).json({ success: true, stock: stockItems });
+    } else {
+      console.log("Sending order to supplier");
+      await fetch(`${supplierApiUrl.suppliers.api_url}`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          ingredientId,
+          bulkOrderQuantity
+        }),
+      });
+      
+    
+    res.status(201).json({ message: "Order successfully sent to Supplier"});
+    }
+  } catch (error) {
+    console.error("Error ordering stock:", error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+stockRouter.post("/supplierconfirmation", async (req: Request, res: Response) => {

```

Comment:
hmmm not sure how we will use this. If the supplier sends us the updated stock, it should be a put / post request to the stock endpoint. This endpoint is very RPC ish, not very RESTful. I'm ok with leaving it but not a good flow.